components/mpas-ocean/bld/namelist_files/namelist_definition_mpaso.xml

```diff
@@ -2046,7 +2046,7 @@ Default: Defined in namelist_defaults.xml
 
 <entry id="config_n_ts_iter" type="integer"
 	category="split_timestep_share" group="split_timestep_share">
-number of large iterations over stages 1-3
+number of large iterations over stages 1-3; For the split_explicit_ab2 time integrator, this value only affects the first time step when it is not a restart run. For restart runs, this value has no effect on the split_explicit_ab2 time integrator.
```

Comment:
@jonbob, this came over when I updated with the `generate_e3sm_namelist_files.py` tool.  Was this manually deleted previously on purpose or just missed in a previous merge?

Reply:
I think it was missed in an earlier merge. I think #6224 will bring it in